doc/controls/ExtendedSplashScreen.md

```diff
@@ -3,30 +3,60 @@ uid: Toolkit.Controls.ExtendedSplashScreen
-| `SplashIsEnabled` | `bool`                 | Gets a value that determines whether native splash is to be displayed for the current platform at runtime. Note for Android 12+ the native splash screen is always disabled irrespective of the Platforms value.                                     |
+| Property | Type | Description | Value |
+|----------|------|-------------|---------------|
+| **Platforms** | `SplashScreenPlatform` | Gets or sets the platform(s) where extended splash screen should be used. This is a flag enumeration, which allows for combining multiple values. | Default value is **All**. Other possible values include **Android**, **iOS**, **Windows**, **WebAssembly**, **Skia**, and **None**. |
+| **SplashIsEnabled** | `bool` | Gets a value representing whether the current environment is to display this splash screen. | **True** if the current platform is included in the **Platforms** property, otherwise **false**. |
+<!-- | **Window** | `Window` | Gets or sets a window for the application that is used to size the splash screen. | -->
+
+## Methods
+
+| Method | Return Type | Description |
+|--------|-------------|-------------|
+| **Init** | `void` | Initializes the splash screen for the provided `Activity` instance. This static method should be invoked from the **OnCreate** override in `MainActivity`.<br/>**Note: This method only needs to be called on Android** |
 
 ## Usage
 
+### Specify platforms to display splash screen
+
+The following code snippet will only display the splash screen on Android and iOS by specifying a `SplashScreenPlatform` value for the `Platforms` property.
+
 ```xml
-xmlns:utu="using:Uno.Toolkit.UI"
+<utu:ExtendedSplashScreen x:Name="Splash"
+                          Platforms="Android,iOS"
 ...
+```
+
+### Set loading state content
+
+The example below demonstrates a typical use of `ExtendedSplashScreen` in XAML. The [**LoadingContentTemplate**](xref:Toolkit.Controls.LoadingView) property below is inherited from `LoadingView`. This property is used to define content that will be displayed during the loading/waiting state.
+
+```xml
+<!-- xmlns:utu="using:Uno.Toolkit.UI" -->
 
-<!-- example -->
 <utu:ExtendedSplashScreen x:Name="Splash"
-                            HorizontalAlignment="Stretch"
-                            VerticalAlignment="Stretch"
-                            HorizontalContentAlignment="Stretch"
-                            VerticalContentAlignment="Stretch"
-                            Platforms="Android,iOS,Windows">
+                          Platforms="Android,iOS"
+                          HorizontalAlignment="Stretch"
+                          VerticalAlignment="Stretch"
+                          HorizontalContentAlignment="Stretch"
+                          VerticalContentAlignment="Stretch">
```

Comment:
`Stretch` isn't the default value for those properties?

Reply:
`Stretch` isn't the default value for those properties?